app/experimenter/nimbus-ui/src/components/PageHome/FilterBar/index.tsx

```diff
@@ -100,7 +100,7 @@ const FilterSelect = <T extends NullableObjectArray>({
   return (
     <Nav.Item
       className="m-1 text-left flex-grow-1 flex-shrink-1"
-      style={{ flexBasis: "0px" }}
+      style={{ flexBasis: "0px", maxWidth: "25%" }}
```

Comment:
This feels kind of grungy. I tried using [a sizing class of `mw-25`](https://getbootstrap.com/docs/4.0/utilities/sizing/), but that seemed to do nothing. Also don't like the arbitrary 25%, since that will need changing if/when we add or remove filter selectors. But, this seemed to clamp down on the expansion without wrecking the responsive layout

Reply:
Looks like `mw-25` doesn't exist; [only](https://github.com/twbs/bootstrap/blob/main/dist/css/bootstrap-utilities.css#L542-L544) `mw-100` exists.

How about we make our own `mw-25` and stick it with the [other](https://github.com/mozilla/experimenter/blob/main/app/experimenter/nimbus-ui/src/styles/index.scss) hand-rolled SCSS? If nothing else but to help keep custom styles class-based

Reply:
Ohhh, I guess I got that confused with `w-25` ugh